### ISSUE TITLE
API Ontology: Missing axiom expiresAt only xsd:dateTime in class Subscription

### ISSUE THREAD

**Issue by nscheiber-champ** [IATA-Cargo/ONE-Record#274]
The API ontology is missing an axiom stating that the property expiresAt only expects xsd:dateTime on class Subscription:

```
[ rdf:type owl:Restriction ;
                owl:onProperty :expiresAt ;
                owl:allValuesFrom xsd:dateTime
                       ] 
```

This leads to code generators generating erratic code and is inconsistent, as other properties expecting xsd:dateTime have that axiom.

**Comment by aloccid-iata** [comment]
HI Niclas,
I think this is how the code library is interpreting the ontology. 
The ontology already have a data range:

                             [ rdf:type owl:Restriction ;
                                owl:onProperty :expiresAt ;
                                owl:maxQualifiedCardinality "1"^^xsd:nonNegativeInteger ;
                                owl:onDataRange xsd:dateTime
                              ]
If needed we can add your axiom in the ontology as an improvement. 

Best,
Davide

**Comment by nscheiber-champ** [comment]
Hi Davide, 
agreed, though many code generators struggle with this type of axiom missing. We compromised in the cargo ontology for that reason, refraining from using "owl:maxQualifiedCardinality" during 3.0.0 dev.

I can add and request to pull?

Cheers
Niclas

**Comment by aloccid-iata** [comment]
Yes fine, I will need to create a jira and link it to the commit
